webapp/channels/src/components/permissions_gates/system_permission_gate/system_permission_gate.tsx

```diff
@@ -25,18 +25,15 @@ type Props = {
     children: React.ReactNode;
 };
 
-export default class SystemPermissionGate extends React.PureComponent<Props> {
-    public static defaultProps = {
-        invert: false,
-    };
-
-    render() {
-        if (this.props.hasPermission && !this.props.invert) {
-            return this.props.children;
-        }
-        if (!this.props.hasPermission && this.props.invert) {
-            return this.props.children;
-        }
-        return null;
+export default function SystemPermissionGate(props: Props) {
```

Comment:
Just for consistency, we prefer to have arrow functions and the export separated at the end. i.e.:
```
const SystemPermissionGate = (props: Props) => {
   ...
}
export default SystemPermissionGate;